File: test/python/unittest-to-pytest/unconvertible/UNCONV-004.expected.py
# HAMLET-TODO [UNCONVERTIBLE-MODULE-SETUP]: Module-level setup/teardown has no direct pytest equivalent in-file
# Original: def tearDownModule():
# Manual action required: Move to conftest.py as a session/module-scoped fixture
def tearDownModule():
    print("module teardown")

def test_example():
    assert True

File: test/java/junit5-to-testng/unconvertible/UNCONV-004.expected.java
import org.junit.jupiter.api.TestFactory;

public class TestFactoryTest {
    // HAMLET-TODO [UNCONVERTIBLE-TEST-FACTORY]: JUnit 5 @TestFactory has no TestNG equivalent
    // Original: @TestFactory
    // Manual action required: Use @DataProvider or @Factory in TestNG
    @TestFactory
    public void testFactory() {
        assert true;
    }
}
